example/switch/switch_run.sh

```diff
@@ -6,33 +6,27 @@
 # SPDX-License-Identifier:     BSD-3-Clause
 #
 
-NUM_RX_PORT=3
 RETVAL=0
 
-PCAP_IN=`find . ${TEST_DIR} $(dirname $0) -name udp64.pcap -print -quit`
-
-echo "Switch test using PCAP_IN = ${PCAP_IN}"
+if  [ -f ./interface_env ]; then
+	. ./interface_env
+else
+        echo "BUG: unable to find interface_env!"
+        echo "interface_env has to be in current directory"
+        exit 1
+fi
 
-RX_PORTS=""
-for i in `seq 1 $NUM_RX_PORT`;
-do
-	RX_PORTS="${RX_PORTS},pcap:out=pcapout${i}.pcap"
-done
+setup_interfaces
 
-./odp_switch${EXEEXT} -i pcap:in=${PCAP_IN}${RX_PORTS} -t 1
+./odp_switch${EXEEXT} -i $IF0$IF1 -t 1
```

Comment:
Maybe it's clearer just to use `$IF0,$IF1,$IF2,$IF3` here.

Reply:
Yeah. I thought so but just to use existing one, I followed this one. I will change in next version.